sample/src/main/kotlin/io/getstream/chat/sample/application/App.kt

```diff
@@ -35,13 +35,14 @@ class App : Application() {
                 androidLogger()
             }
             androidContext(this@App)
-            modules(
-                listOf(
+
+            // see crash/bug here: https://github.com/InsertKoinIO/koin/issues/871
+            koin.loadModules(listOf(
                     appModule,
                     dataModule,
                     loginModule
-                )
-            )
+            ))
+            koin.createRootScope()
```

Comment:
I think we can move forward getting rid of koin in sample app. It turns out we use mostly singleton dependencies and don't need any advanced DI features. This will also make sample app's code more independent of external libs - easier to consume by users.